pkg/webhook/server.go

```diff
@@ -31,8 +31,8 @@ import (
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"sigs.k8s.io/controller-runtime/pkg/internal/certwatcher"
```

Comment:
I believe that imports are intentionally grouped together such that all `sigs.k8s.io/controller-runtime` would be separate.

Reply:
I'm not sure, saw many locations where the `sigs.k8s.io/controller-runtime` was imported from other packages, and duplicating the code here seemed bad to me. Could do this if you consider that it's necessary.